test/api/getLedger/index.ts

```diff
@@ -0,0 +1,78 @@
+import assert from 'assert-diff'
+import { assertResult, TestSuite } from '../utils'
+import responses from '../../fixtures/responses'
+const { getLedger: RESPONSE_FIXTURES } = responses
+
+export default {
+  'simple test': async api => {
```

Comment:
Would these test cases become more indexable if you used an enum rather than a magic string?  What about an array that you could just iterate over?

Reply:
Can you explain a bit more about what you mean by more indexable? I'd say that this was designed primarily for the user experience flow and was designed to match what it feels like to write tests in mocha (string -> test function). My worry is that an array would feel odd as a test writer/maintainer.

Reply:
I guess I assumed you may share these magic strings across tests, which might  make them better as an emum.

As a programmer, I'd still expect this class to be an array (of objects which contained a title, and a test function) but I'm fine letting this go in if you think it's idiomatic for javascript.

Reply:
Sounds good 👍 Happy to revisit later as well if we think there's a better way